lib/coproc.cpp

```diff
@@ -48,6 +43,12 @@
 #include "str_replace.h"
 #include "util.h"
 
+#ifdef _USING_FCGI_
```

Comment:
Previously this piece of code was for non '_WIN32' only.
Could you please keep it?